package.json

```diff
@@ -16,8 +16,9 @@
   },
   "scripts": {
     "lint": "eslint index.js",
+    "closure-compiler-check": "google-closure-compiler -O ADVANCED --generate_exports --js_output_file /dev/null --jscomp_error '*' closure-compiler-check/* index.js",
```

Comment:
Is there a reason to use `ADVANCED` or `--generate_exports` when not using output file?  Why not use `-O SIMPLE --checks_only`?

Reply:
Good question.  I was initially using `ADVANCED` and `generate_exports` because that's what Shaka Player uses, and I was examining the output to make sure that exports were correctly generated.  I had to add `closure-compiler-check/goog-stub.js` to make `generate_exports` work without the closure library.  I could probably remove both the stubs and generate_exports, but it would be convenient to be able to examine the output sometimes before releasing a new version.  If I only have to remove --js_output_file /dev/null, that's preferable to trying to remember why it fails when I add `generate_exports` and recreating the stubs, etc.

What do you think?